erasure-readfile.go

```diff
@@ -90,9 +90,10 @@ func (e erasure) ReadFile(volume, path string, startOffset int64, buffer []byte)
 
 	// Copy data blocks.
 	copy(buffer, dataBlocks)
+	n := int64(len(dataBlocks))
 
 	// Relenquish memory.
 	dataBlocks = nil
```

Comment:
Setting it to nil here is of no use. dataBlocks is losing scope here and it will be garbage collected anyway. Modify getDataBlocks to take buffer as arg and avoid copying in the first place.